docs/copilot/copilot-chat.md

```diff
@@ -1,7 +1,7 @@
 ---
-Order: 3
+Order: 6
 Area: copilot
-TOCTitle: Copilot Chat
+TOCTitle: Chat Conversations
```

Comment:
Hmm I'm actually okay with this being titled `Copilot Chat` - was there something about `Chat Conversations` that resonated more with you @ntrogh (or maybe feedback I'm forgetting that I shared 😄)?

Reply:
I think `Chat` or `Copilot Chat` sound good to me. I might be more inclined to choose `Copilot Chat` since it feels like it's beneficial to keep aligned with any term that might be a "brand" (similar thinking when we renamed this node from "Artificial Intelligence" to "GitHub Copilot").